src/routes/cart.routes.ts

```diff
@@ -15,6 +22,12 @@ cartRouter.get("/", [protectRoute], viewCart);
 
 cartRouter.delete("/remove/:id", [protectRoute], removeFromCart);
 
-cartRouter.delete("/clear", [protectRoute], clearCart);
```

Comment:
@musllim I think you are wrong; 
The is you are telling to make changes to the line of codes write by someone else in team. 
try to focus on my work not on other because this PR is about buyer to update a cart.